ui/imports/shared/views/chat/MessageContextMenuView.qml

```diff
@@ -47,6 +48,27 @@ StatusMenu {
     property bool pinnedMessage: false
     property bool canPin: false
 
+    readonly property bool isNotEmpty: emojiContainer.visible ||
```

Comment:
i don't like this approach but it's the fastest way to get this merged asap

Reply:
Menu has a `count` property, would it work to check 0 instead? Otherwise, maybe some of our QML experts knows a cleaner approach.

Reply:
Yup, sounds much better for sure :)

Reply:
Unfortunately `count` is total, including disabled & invisible. So it's always 25 for me. I'll try to add new prop to StatusMenu for counting menu items

Reply:
He-he
<img width="797" alt="Screenshot 2023-03-21 at 14 48 44" src="https://user-images.githubusercontent.com/2522130/226545198-89ea8846-a79e-4dc8-b412-483e40b27d02.png">